ext/nnstreamer/tensor_filter/tensor_filter_tensorflow.c

```diff
@@ -77,8 +78,12 @@ tf_loadModelFile (const GstTensorFilterProperties * prop, void **private_data)
   tf = g_new0 (tf_data, 1); /** initialize tf Fill Zero! */
   *private_data = tf;
   tf->tf_private_data = tf_core_new (prop->model_file);
+
+  const gboolean tf_mem_optmz =
```

Comment:
IMO, for the purpose of generalization, how about define a struct-typed option container in the layer of tensor_filter. We can use such container in any of sub-plugins of tensor_filter by inheriting (I mean c-style inheritance) it. This would make easier to pass options as function parameters.

Reply:
@wooksong ```nnsconf_get_value_bool ``` is the API for such "struct-typed" option container. As we do not have terribly many options with hierarchical structure, we don't need inheritances, yet.